doc/source/batch-inference/doc_code/torch_image_batch_pretrained.py

```diff
@@ -0,0 +1,51 @@
+from typing import Dict, List
+import numpy as np
+
+import torch
+from torchvision import transforms
+from torchvision.models import resnet18
+
+import ray
+from ray.train.torch import TorchCheckpoint, TorchPredictor
+from ray.train.batch_predictor import BatchPredictor
+from ray.data.preprocessors import TorchVisionPreprocessor
+
+
+data_url = "s3://anonymous@air-example-data-2/1G-image-data-synthetic-raw"
+print(f"Running GPU batch prediction with 1GB data from {data_url}")
+dataset = ray.data.read_images(data_url, size=(256, 256)).limit(10)
+
+transform = transforms.Compose(
+    [
+        transforms.ToTensor(),
+        transforms.CenterCrop(224),
+        transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
+    ]
```

Comment:
Can you explain these magic numbers? Where do the come from?

Reply:
updated